### PR TITLE
No files in this

### PR DESCRIPTION
**Pull request by skthinks**

### CODE REVIEW

File: movie.sh [hunk 1]
```diff
@@ -30,38 +27,33 @@ function main(){
     # There should be exactly one argument i.e the address path
     if [ $NUM_ARGUMENTS != 1 ]; then
         add_style "This application takes one argument : Movie Directory" 1
-        rm -rf $TEMP_PATH
         exit 4
```
Comment (vaidik) [inline movie.sh:30]: Why 4?

File: movie.sh [hunk 2]
```diff
@@ -30,38 +27,33 @@ function main(){
...
     fi
     # Fetching Movies and removing 
-    ls $DIRECTORY_NAME > "$TEMP_PATH/movlist.txt"
-    local movie_count=$(cat "$TEMP_PATH/movlist.txt" \
+    movie_list=$(ls $DIRECTORY_NAME \
+     | grep "" | \
```
Comment (vaidik) [inline movie.sh]: The pipe at the end of the line goes in the next line.

File: movie.sh (hunk 1 above)
Comment (skthinks) [inline movie.sh:30]: I read through the exit codes 
1,2,125,126 etc were reserved
I did not find 4 as reserved so I chose it randomly

File: movie.sh [hunk 3]
```diff
@@ -30,38 +27,33 @@ function main(){
...
-        echo $rating '      '  $movie_name >> "$TEMP_PATH/out.txt"
-    done < "$TEMP_PATH/clean_mov.txt"
-    clear
+        output="$output $rating       $movie_name\n"
+    done
     # Above, a file is created with 2 coloumns of Movie and rating
     # Then it is sorted
-    sort -k 1 -r "$TEMP_PATH/out.txt"
+    echo -e $output | sort -k 1 -r 
     tput setaf 1
```
Comment (vaidik) [inline movie.sh]: I think you can use your `add_style` function here

File: movie.sh (hunk 2 above)
Comment (vaidik) [inline movie.sh]: You started your function with 4 space indentation. Every time you move another level, you add another 4 spaces. So this should be rewritten as:

```
    movie_list=$(ls $DIRECTORY_NAME \
        | grep "" \
        | sed 's/_/./g; s/ /./g')
```

**Pull request merged**